server/catgenome/src/main/java/com/epam/catgenome/manager/blast/BlastTaskManager.java

```diff
@@ -93,28 +96,48 @@ private void loadTaskParameters(final BlastTask blastTask) {
     }
 
     @Transactional(propagation = Propagation.REQUIRED)
-    public BlastTask create(final BlastTask blastTask) throws BlastRequestException {
+    public BlastTask create(final TaskVO taskVO) throws BlastRequestException {
+        if ((taskVO.getOrganisms() == null || taskVO.getOrganisms().isEmpty())
```

Comment:
We can use `CollectionUtils.isEmpty()` method, it is null-safe. And it'd suggest to move this condition to `Assert` clause, e.g.
```
Assert.isFalse(CollectionUtils.isEmpty(taskVO.getOrganisms()) && CollectionUtils.isEmpty(taskVO. getExcludedOrganisms()), message)
```